source/configuration_manual/event_filter.rst

```diff
@@ -106,6 +106,16 @@ For example::
     (event=abc AND (message_size>1gb and message_size<1tB)) OR \
     (event=def AND (duration<1mins))
 
+.. versionchanged:: v2.4.0;v3.0.0 Event fields have specific types that
+                    constrain the possible values they can be filtered by. For
+                    example ``bytes_out`` and ``message_size`` are numeric and
+                    can only be matched against numeric values. Other types are
+                    for example strings/stringlists or timevalues. Until now
```

Comment:
Looks like just above these examples there's "There are some limitations on which operators work with what field types" which lists the types and operators that are supported. Maybe this text belongs right after that list? Then there's no real need to say "Other types are for example strings/stringlists or timevalues" since they were just listed. In any case "timestamp" is the right word for "timevalue".